### PR TITLE
saga-5.136: Modularizing fonts in a separate file

### PR DESCRIPTION
**Pull request by juligobbi-fluxit**
Modularizing fonts in a separate file and import them to the theme. So that in maven it can be imported separately.

_Example in maven:_
```
import { Colors, Fonts, P, Dropdown } from '@trova/components';

export const CardTitle = styled(P)({
        fontFamily: Fonts.fonts.robotoBold,
        fontSize: Fonts.fontSizes.h4,
    },);
```

....................
_*Notes:_
On this task, I made a simple change, just separating the font file from the theme, but there are a few things to do that could be good improvements to take.
I would like to take this deeper, after talking with you guys, about the next steps.

My ideas are:
-Refactoring **font** file (cleaning variables of fontSizes, removing px, adding lineHeights, etc, and maybe here it would be needed to change the path)
-Put **breakpoints** in a separate file (like fonts)
-Adding a **spacing** file
-And import all of them into the theme as I did with Fonts
-Maintain **theme** clean, to have here just the future themes, and their imports files. e.g:

```
theme1: {
...Fonts,
} 
theme2: {
} 
``` 
https://asanal.ink/21/136

### CODE REVIEW

File: src/shared/constants/fonts.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const Fonts = {
```
Comment (BrandonDenham) [inline src/shared/constants/fonts.tsx:1]: This filename should just be Ts as it is not a react component

File: src/shared/constants/index.tsx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { Colors } from './colors';
```
Comment (BrandonDenham) [inline src/shared/constants/index.tsx:1]: same here, this  should be .ts not .tsx

.tsx is reserved when exporting a react component.

**Review by BrandonDenham — APPROVED** [review]
minor comments

File: src/shared/constants/fonts.tsx (hunk 1 above)
Comment (juligobbi-fluxit) [reply src/shared/constants/fonts.tsx:1]: of course, thanks for that!

**Review by rmcarias — APPROVED** [review]

**Pull request merged**